data.rkt

```diff
@@ -114,9 +114,13 @@
 (define speaker-id-lam                      "lam")
 (define speaker-id-jim-roucis               "jim-roucis")
 (define speaker-id-jared                    "jared")
+(define speaker-id-tigran-khandkar          "tigran-khandkar")
+(define speaker-id-jack-jutzi               "jack-jutzi")
+(define speaker-id-lisa-???                 "lisa-???")
+(define speaker-id-daniel-krol              "daniel-krol")
 
-(define host-id-raven-labs            "raven-labs")
-(define host-id-manchester-makerspace "manchester-makerspace")
+(define host-id-raven-labs                  "raven-labs")
+(define host-id-manchester-makerspace       "manchester-makerspace")
```

Comment:
This is unrelated to the PR - these IDs did not change.

Reply:
Should this spacing fix be removed, then?

Reply:
Yes